frontend/src/pages/add-blog.tsx

```diff
@@ -58,11 +74,11 @@ function AddBlog() {
       return false;
     }
 
-    const imageLinkRegex = /\.(jpg|jpeg|png)$/i;
-    if (!imageLinkRegex.test(formData.imageLink)) {
-      toast.error('Image URL must end with .jpg, .jpeg, or .png');
-      return false;
-    }
+    // const imageLinkRegex = /\.(jpg|jpeg|png)$/i;
+    // if (!imageLinkRegex.test(formData.imageLink)) {
+    //   toast.error('Image URL must end with .jpg, .jpeg, or .png');
+    //   return false;
+    // }
     if (formData.categories.length > 3) {
```

Comment:
Why are we commenting this ? can we instead add WebP in this regex

Reply:
Added :)